src/applications/mhv/medications/actions/prescriptions.js

```diff
@@ -39,15 +43,45 @@ export const clearAllergiesError = () => async dispatch => {
 };
 
 export const getPrescriptionDetails = prescriptionId => async dispatch => {
-  const response = await getPrescription(prescriptionId);
-  const prescription = response.data.attributes;
-  dispatch({ type: Actions.Prescriptions.GET_DETAILS, prescription });
+  try {
+    const response = await getPrescription(prescriptionId);
+    const prescription = response.data.attributes;
+    dispatch({ type: Actions.Prescriptions.GET_DETAILS, prescription });
+    return null;
+  } catch (error) {
+    dispatch({
+      type: Actions.Prescriptions.GET_API_ERROR,
+    });
+    return error;
+  }
 };
 
 export const setPrescriptionDetails = prescription => async dispatch => {
   dispatch({ type: Actions.Prescriptions.SET_DETAILS, prescription });
 };
 
+export const getRefillablePrescriptionsList = (
+  pageNumber,
+  sortEndpoint,
+) => async dispatch => {
+  try {
+    const response = await getRefillablePrescriptionList(
+      pageNumber,
```

Comment:
there are no params for this function, so no need to pass in pageNumber or sortEndpoint

```
export const getRefillablePrescriptionList = () => {
  return apiRequest(
    `${apiBasePath}/prescriptions/list_refillable_prescriptions`,
    { headers },
  );
};
```